src/app/rosetta/README.md

```diff
@@ -0,0 +1,10 @@
+Implementation of the [Rosetta Node API](https://www.rosetta-api.org/docs/node_api_introduction.html) for Coda.
+
+To regenerate the models:
+```
+git clone https://github.com/coinbase/rosetta-specifications.git
+cd rosetta-specifications
+`brew install openapi`
```

Comment:
is this Mac-specific?

Reply:
The instructions are but you should be able to install the openapi tool wherever. I’ll try to scare up some Linux install instructions